script.speedfaninfo/addon.xml

```diff
@@ -0,0 +1,88 @@
+		<news>
+v.1.0.0
+- Python 2/3 compatibility
+- converted language files to current format for forward compatability
+- fix for addon not quitting properly if running when Kodi attempts to exit
+- updated for new method of retrieving preferences
+- updated icon
+		</news>
+		<assets>
+        	<icon>icon.png</icon>
+        </assets>
+		<summary lang="af_ZA">Vertoon inligting van jou SpeedFan Aantekenings.</summary>
+		<summary lang="be_BY">Display information from your SpeedFan log.</summary>
+		<summary lang="bg_BG">Показва информация от вашия SpeedFan дневник.</summary>
+		<summary lang="ca_ES">Mostra la informació del Registre de Velocitats del Ventilador.</summary>
+		<summary lang="da_DK">Vis oplysninger fra din SpeedFan-log.</summary>
+		<summary lang="de_DE">Zeige Informationen aus deinem SpeedFan Log.</summary>
+		<summary lang="el_GR">Προβολή πληροφοριών από το αρχείο καταγραφής του SpeedFan.</summary>
+		<summary lang="en_GB">Display information from your SpeedFan log.</summary>
+		<summary lang="en_NZ">Display information from your SpeedFan log.</summary>
+		<summary lang="en_US">Display information from your SpeedFan log.</summary>
+		<summary lang="es_ES">Muestra información del log de SpeedFan</summary>
+		<summary lang="es_MX">Muestra información desde tu log de SpeedFan</summary>
+		<summary lang="et_EE">Kuvab info SpeedFan logist.</summary>
+		<summary lang="fr_FR">Afficher les informations à partir du journal de SpeedFan</summary>
+		<summary lang="fr_CA">Afficher l'information du journal pour SpeedFan</summary>
+		<summary lang="gl_ES">Amosar información do teu rexistro de SpeedFan.</summary>
+		<summary lang="he_IL">הצג מידע מיומן הרישום של SpeedFan.</summary>
+		<summary lang="hr_HR">Prikaz informacija iz vašeg SpeedFan zapisa.</summary>
+		<summary lang="hu_HU">Ventilátor-sebesség napló megjelenítése</summary>
+		<summary lang="id_ID">Menampilkan informasi log SpeedFan Anda.</summary>
+		<summary lang="it_IT">Mostra le informazioni dal tuo log di SpeedFan</summary>
+		<summary lang="nl_NL">Laat informatie zien van het SpeedFan log</summary>
+		<summary lang="pl_PL">Wyświetla infromacje z dziennika SpeedFan</summary>
+		<summary lang="pt_PT">Exibe informação do seu log do Speedfan.</summary>
+		<summary lang="pt_BR">Mostrar informações de seu registro SpeedFan.</summary>
+		<summary lang="ru_RU">Отображает информацию из Вашего журнала SpeedFan.</summary>
+		<summary lang="sk_SK">Zobraziť informácie z Vášho SpeedFan logu.</summary>
+		<summary lang="sv_SE">Visar information från din SpeedFan logg.</summary>
+		<summary lang="zh_CN">显示来源于SpeedFan日志的信息</summary>
+		<description lang="af_ZA">Ontleed en vertoon inligting van die SpeedFan Aantekenings. Terwyl hierdie plugin werk op enige platform, SpeedFan werk net op Windows, so dit is optimaal gebou vir mense wat XBMC op Windows gebruik. Sien readme.txt vir inligting oor hoe om jou SpeedFan Aantekenings in te stel sodat die prop dit behoorlik kan ontleed nie.</description>
```

Comment:
consider to change XBMC to Kodi in these strings in a future update :)

Reply:
I caught the one in the readme (which of course nobody sees), but didn't see those.  One of these days I'll find all the instances of XBMC in things and change them.  '-)